regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -107,6 +108,10 @@ var CommentReviewView = Backbone.View.extend({
     this.submitMode();
   },
 
+  back: function() {
+    window.history.back();
+  },
```

Comment:
I just realized what you probably meant which was to directly pass the function. That doesn't work either because it will immediately call it (so once you reach review page it slingshots you back) rather than to wait when this.back() is invoked.